book/website/collaboration/leadership/leadership-opportunities.md

```diff
@@ -0,0 +1,59 @@
+You can explore the chapter in detail, which spotlights the roles of Research Community Managers, Research Applications Managers, Research Engineers, Data Stewards, Data Wranglers, Project managers and more such roles as open leaders.
+
+For community projects, The Carpentries' community leadership documentation in [The Carpentries' Handbook](https://docs.carpentries.org/topic_folders/governance/index.html) make a good set of resource.
+
+## Onboarding, Nurturing and Offboarding Gracefully
+
+### Onboarding new leaders
+
+Onboarding leaders is a process through which new members join an organisation at a leadership position.
+Onboarding occurs after a new member has agreed to join the organisation upon invitation by the current leadership team or via a formal election process.
+A set of guidelines can be provided in a written format as a formal or informal document to ensure that new members have the opportunity to familiarise themselves with the process and resources they can use in their work, as well as people in the organisation they can collaborate with. 
+The process is particularly important to make the new members or leaders feel welcome in their organisation.
+Here are a few steps that may be involved in the onboarding process:
+- Updating internal and external documents to communicate the new position or active status of newly onboarded members with the appropriate party
+- Documents with details on organisation policy, project details and job responsibilities
+- Giving access to platforms and documents to ensure required/permitted details are transparently shared
+- Roles and responsibilities of other members and leaders in the organisation
+- Meeting scheduled with all stakeholders in the project with who the newly onboarded member can collaborate with
+
+### Nurturing the next leaders
+
+In data science and research, leaders are often too busy leading (admin tasks, reporting, funding and project meetings) and hence leaders rarely have time to learn about the latest advances both at technical and human levels. 
+Therefore, leaders need to allow members of their team to work independently, while also supporting them in exchanging knowledge among themselves.
+Senior leaders should commit to building diverse teams with members and leaders from various backgrounds and levels of experience. 
+It is then, their responsibility to mentor and support them and create a path for them to excel and grow in their roles. 
+
+Nurturing these next-generation leaders requires building trust through authentic connections, and providing opportunities for upskilling.
+Providing the right level of support while challenging and nudging their team members towards new directions helps them upskill and experience personal growth.
```

Comment:
```suggestion
Providing the right level of support while challenging and nudging their team members towards new directions helps **them** upskill and experience personal growth.
```
Is this "them" the leader or the team members? Not clear

Reply:
Team :)